.docs/Linux.md

```diff
@@ -14,7 +14,10 @@ Please follow the [instructions provided here](https://github.com/Tyrrrz/Discord
 ```console
 dotnet DiscordChatExporter.Cli.dll export -t TOKEN -c CHANNEL
 ```
-
+If the above command throws a "Permission denied" error, execute this command to fix the permissions:
+```console
+chmod 644 *.dll DiscordChatExporter.*
+```
```

Comment:
```suggestion

If the above command throws a "Permission denied" error, execute this command to fix the permissions:

```console
chmod 644 *.dll DiscordChatExporter.*
```

```

Reply:
You accidentally deleted the closing backticks for the code block which messed up the formatting of the text below.

![image](https://github.com/Tyrrrz/DiscordChatExporter/assets/45744247/09f0aa92-4823-46e9-a5f6-ce439fa925e5)

Reply:
Sorry, I must've confused them for the backticks matching the `suggestion` block. Will fix it on master.